test.py

```diff
@@ -34,8 +34,12 @@ def _notebook_run(path):
 def run_tests():
     doctest.testmod(utils.riemann_tools)
 
+    # notebooks to ignore in tests:
+    ignore_notebooks = ['Make_html_animations.ipynb']
```

Comment:
Why do we not test this notebook?

Reply:
It's just run to make animations to post on the webpage for the html version, not needed by users.  It's fine to test too I guess.